packages/maker/dmg/package.json

```diff
@@ -7,6 +7,10 @@
   "license": "MIT",
   "main": "dist/MakerDMG.js",
   "typings": "dist/MakerDMG.d.ts",
+  "scripts": {
+    "test": "yarn test:base test/**/*_spec.ts",
+    "test:base": "cross-env TS_NODE_FILES=1 mocha --config ../../../.mocharc.js"
+  },
```

Comment:
This isn't directly related. I just needed an easy way to run just the DMG maker's tests.